features/datum.feature

```diff
@@ -331,85 +326,81 @@ Functionaliteit: leveren van een datum
+      | 'partner' | partner    | partners    | inOnderzoek          | datumIngangOnderzoek | Datum          | 050000                 | 20200308 | 2020-03-08 |      |       |          | 8 maart 2020 |
+      | 'partner' | partner    | partners    | inOnderzoek          | datumIngangOnderzoek | DatumOnbekend  | 050000                 | 00000000 |            |      |       | true     | onbekend     |
+      | 'partner' | partner    | partners    | inOnderzoek          | datumIngangOnderzoek | JaarDatum      | 050000                 | 20200000 |            | 2020 |       |          | 2020         |
+      | 'partner' | partner    | partners    | inOnderzoek          | datumIngangOnderzoek | JaarMaandDatum | 050000                 | 20200300 |            | 2020 | 3     |          | maart 2020   |
+      | 'kind'    | kind       | kinderen    | inOnderzoek          | datumIngangOnderzoek | Datum          | 090000                 | 20200308 | 2020-03-08 |      |       |          | 8 maart 2020 |
+      | 'kind'    | kind       | kinderen    | inOnderzoek          | datumIngangOnderzoek | DatumOnbekend  | 090000                 | 00000000 |            |      |       | true     | onbekend     |
+      | 'kind'    | kind       | kinderen    | inOnderzoek          | datumIngangOnderzoek | JaarDatum      | 090000                 | 20200000 |            | 2020 |       |          | 2020         |
+      | 'kind'    | kind       | kinderen    | inOnderzoek          | datumIngangOnderzoek | JaarMaandDatum | 090000                 | 20200300 |            | 2020 | 3     |          | maart 2020   |
 
   Rule: verblijfplaats datumVan wordt gevuld uit datumAanvangAdreshouding of datumAanvangAdresBuitenland
 
     Abstract Scenario: datum aanvang adreshouding (10.30) <titel>
-      Gegeven het systeem heeft een persoon met de volgende gegevens
-      | naam                | waarde    |
-      | burgerservicenummer | 555550001 |
-      En de persoon heeft de volgende 'verblijfplaats' gegevens
+    Gegeven de persoon met burgerservicenummer '000000267' heeft de volgende 'verblijfplaats' gegevens
       | naam                               | waarde                     |
-      | straatnaam (11.10)                 | een straat                 |
+      | functieAdres.code (10.10)          | W                          |
       | datum aanvang adreshouding (10.30) | <datumAanvangAdreshouding> |
       Als personen wordt gezocht met de volgende parameters
       | naam                | waarde                          |
       | type                | RaadpleegMetBurgerservicenummer |
-      | burgerservicenummer | 555550001                       |
-      | fields              | verblijfplaats.datumVan         |
+      | burgerservicenummer | 000000267                       |
+      | fields              | verblijfplaats         |
```

Comment:
waarom is datumVan uit de fields verwijderd? Dit betekent namelijk dat de hele verblijfplaats wordt gevraagd en moet functieAdres veld ook in de Dan stap worden toegevoegd

Reply:
datumVan weer toegevoegd. Was verwijderd tijdens de zoektocht om het scenario aan de praat te krijgen..... Had hem terug moeten zetten.